sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -84,7 +84,13 @@ def _safe_issubclass(derived, parent):
     issubclass(derived, parent), or False if a TypeError was raised.
   """
   try:
-    return issubclass(derived, parent)
+    if hasattr(typing, '_GenericAlias') and \
+      (type(derived) == typing._GenericAlias):
```

Comment:
Could we please use parens to avoid adding a backslash (see PEP8P)?
```python
    if (hasattr(typing, '_GenericAlias') and
        type(derived) == typing._GenericAlias):
```
One whitespace character to the right of the backslash and the script breaks on a change that is invisible to the reader.